pyproject.toml

```
@@ -28,6 +28,28 @@ dependencies = [
     "shapely >= 2.0.0",
 ]
 
+[project.optional-dependencies]
+geodatabase = [
```

I think we should try and align with pandas which has options for `postgresql`, `mysql`, `sql-other`, hence postgis (we could have a generic `sql` but it would only have one package in it).
```suggestion
postgis = [
```